integrations/magnitude.mdx

```diff
@@ -43,10 +48,9 @@ const agent = await startBrowserAgent({
     browser: {
         cdp: kernelBrowser.cdp_ws_url,
         contextOptions: {
-            viewport: { width: 1024, height: 768 }
-        }
+            viewport: { width: 1920, height: 1080 }
+    }
```

Comment:
@tembo can you fix the formatting here, so the closing "}" is aligned with contextOptions